plugins/auth-backend/src/providers/saml/provider.ts

```diff
@@ -172,7 +172,7 @@ export const saml = createAuthProviderIntegration({
         logoutUrl: config.getOptionalString('logoutUrl'),
         audience: config.getString('audience'),
         issuer: config.getString('issuer'),
-        cert: config.getString('cert'),
+        idpCert: config.getString('cert'),
```

Comment:
The constructor argument types changed as part of https://github.com/node-saml/passport-saml/pull/894